blablacar-persistence/src/main/java/cz/fi/muni/pa165/teamred/entity/User.java

```diff
@@ -12,6 +12,7 @@
  * Created by Jozef Cibík on 25.10.2017.
  */
 @Entity
+@Table(name = "users")
 @Inheritance(strategy = InheritanceType.JOINED)
 public class User implements Serializable{
```

Comment:
Chceme tu to serializable? Ja by som to asi dal prec :)

Reply:
To tam uz bolo predtym, to som nemenil. Ale pridal by som to tam len ak to potrebujeme, KISS princip.

Reply:
jj, tak mozes vramci tohto PR vyhodit